### PR TITLE
Feat: Lido Multichain

### PR DESCRIPTION
**Pull request by TheDZhon**
Just rename all of the `Lido on L2` mentions to `Lido Multichain` to follow the up-to-date branding approach.

### CODE REVIEW

File: docs/token-guides/wsteth-bridging-guide.md [hunk 1]
```diff
@@ -377,7 +377,7 @@ In the comments section, please provide the relevant details: the artifacts, if
 
 ## References
 
-- Deployed contracts addresses https://docs.lido.fi/deployed-contracts/#lido-on-l2
+- Deployed contracts addresses https://docs.lido.fi/deployed-contracts/#lido-multichain
 - LOL (Liquidity Observation Labs) https://research.lido.fi/t/liquidity-observation-lab-lol-liquidity-strategy-and-application-to-curve-steth-eth-pool/5335
 - Lido L2 reference bridging contracts (Arbitrum and Optimism) https://github.com/lidofinance/lido-l2
```
Comment (karinamaulitova) [inline docs/token-guides/wsteth-bridging-guide.md:382]: Should we rename this as well?

Comment (TheDZhon) [reply docs/token-guides/wsteth-bridging-guide.md:382]: I believe we should, why not?

**Review by grstepanov — APPROVED** [review]

File: docs/token-guides/wsteth-bridging-guide.md (hunk 1 above)
Comment (folkyatina) [reply docs/token-guides/wsteth-bridging-guide.md:382]: Not sure. Multichain includes other L1s, but this project intended exclusively for L2. So, I'd rather not

**Review by folkyatina — APPROVED** [review]

File: docs/token-guides/wsteth-bridging-guide.md [hunk 2]
```diff
@@ -43,7 +43,7 @@ Lido DAO can recognize the bridged wstETH endpoints by means of a signalling sna
```
Comment (tamtamchik) [inline docs/token-guides/wsteth-bridging-guide.md:43]: Line 41: add trailing comma 
Line 42: `Mode>` ⇒ `Mode`?

**Review by tamtamchik — APPROVED** [review]

File: docs/token-guides/wsteth-bridging-guide.md (hunk 1 above)
Comment (TheDZhon) [reply docs/token-guides/wsteth-bridging-guide.md:382]: Agh, I misread it, sorry; it should stay the same bc it's a project name for the particular Arbi/Opti bridges ofc :)

File: docs/token-guides/wsteth-bridging-guide.md (hunk 2 above)
Comment (TheDZhon) [reply docs/token-guides/wsteth-bridging-guide.md:43]: Gotcha!

**Pull request merged**